bridge/standard/pkg/node/node.go

```diff
@@ -100,7 +100,7 @@ func NewNode(opts *Options) (*Node, error) {
 		ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
 		defer cancel()
 
-		client, err := ethclient.DialContext(ctx, opts.L1RPCURL)
+		client, err := ethclient.DialContext(ctx, opts.L1RPCURLs[0])
```

Comment:
What is the point to submit an array, if we just take the first value?

Reply:
Oh, we assume, that first one will work and in wrapped client use the array

Reply:
Yes, technically all work. If the first one doesnt work, we will fail in the initialization itself. This is mostly needed if we get rate-limited or if the RPC is down at some later point.

For the initial call we only do it to get the chain-id. We dont configure it in the CLI.